x-pack/plugins/observability_solution/infra/public/plugin.ts

```diff
@@ -148,7 +159,7 @@ export class Plugin implements InfraClientPluginClass {
                     label: logsTitle,
                     sortKey: 200,
                     entries: getLogsNavigationEntries({
-                      application,
+                      isLogsExplorerAccessible: isLogsExplorerAccessible as boolean,
```

Comment:
If we convert the value to a boolean as suggested below we don't need this cast.

```suggestion
                      isLogsExplorerAccessible,
```